helm/adelphi/templates/gemini-job.yaml

```diff
@@ -47,6 +48,21 @@ spec:
       volumeMounts:
       - name: workspace
         mountPath: /workspace
+    - name: extract-schema
+      image: python:3.10.0a1-alpine3.12
```

Comment:
@absurdfarce fyi, I replaced the original image you had here with the official one for Python.